File: server/src/main/scala/kpn/api/common/planner/PlanFragment.scala
package kpn.api.common.planner

import kpn.api.common.LatLonImpl

case class PlanFragment(
  meters: Long,
  orientation: Long,
  streetIndex: Option[Long],
  coordinate: PlanCoordinate,
  latLon: LatLonImpl
) {
}

File: backend/api/src/main/java/kpn/api/common/planner/PlanFragment.java
package kpn.api.common.planner;

import kpn.api.common.LatLonImpl;
import kpn.api.common.planner.PlanCoordinate;

public record PlanFragment(
  Long meters,
  PlanCoordinate coordinate,
  LatLonImpl latLon
) {
}
